modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/info/collection/provider/SXPBlueprintInfoCollectionProvider.java

```diff
@@ -103,7 +104,7 @@ public InfoPage<AssetEntry> getCollectionInfoPage(
 						sxpBlueprintExternalReferenceCode,
 						serviceContext.getCompanyId());
 
-			Pagination pagination = collectionQuery.getPagination();
+			// TODO Set default from and size values if pagination is null
```

Comment:
@ruben-pulido Can pagination be null?

```
	public Pagination getPagination() {
		if (_pagination == null) {
			return Pagination.of(20, 0);
		}

		return _pagination;
	}
```

Reply:
note: maybe the default Pagination should consider the `search.container.page.default.delta` portal property. (Which defaults to 20, though it can change.